pkgs/development/libraries/gpgme/default.nix

```diff
@@ -31,6 +31,14 @@ stdenv.mkDerivation rec {
   nativeBuildInputs = [ file pkgconfig gnupg autoreconfHook git texinfo ]
   ++ lib.optionals pythonSupport [ python swig2 which ncurses ];
 
+  patches = [
+    (fetchpatch {
+      name = "fix-key-expiry.patch";
+      url = "https://files.gnupg.net/file/data/fehgbjmataj5tc2pnfhj/PHID-FILE-aqck6l4elhw53tjanrie/file";
```

Comment:
```404 Not Found``` as by now

Reply:
Hmm, I think this might be stable: https://dev.gnupg.org/rM66376f3e206a1aa791d712fb8577bb3490268f60?diff=1

Reply:
or "https://git.gnupg.org/cgi-bin/gitweb.cgi?p=gpgme.git;a=patch;h=66376f3e206a1aa791d712fb8577bb3490268f60" (hash different because less number of unaffected lines are included in the patch)

Reply:
Am I missing a reason why no one actually changed this (it would e.g. cause a few rebuilds but IMO it would definitely make sense, not sure if everyone uses https://hydra.nixos.org)? If not I would suggest we use the URL from @volth (as the patch is more compact and this commit is from the master branch). If no one changes this and/or responds I'll change it in a couple of hours (before we forget about it and it causes some issues (cache misses, etc.)).

Reply:
> Am I missing a reason why no one actually changed this 

I assume neither @hedning nor me have write access and it seemed to small an issue to make a PR, so it wasn just left for a passing-by nixos member :)

Reply:
Ok, thanks for the quick response, I've updated the URL in 399761ea0a9cf1f32556e5cf55e613d319761a45.